_posts/2017-10-09-omero-5-4-0.md

```diff
@@ -0,0 +1,87 @@
+---
+Today we are releasing OMERO 5.4.0, a full production-ready release of OMERO;
+featuring a new configurable user role "Restricted Administrators"; further
+improvements to OMERO.web; additions to OMERO.cli; and many fixes and
+performance improvements:
+
+-  added [Administrators with restricted privileges](https://docs.openmicroscopy.org/omero/5.4.0/sysadmins/admins-with-restricted-privileges.html) to allow sysadmins to delegate
+   management tasks to facility managers without granting them full system
+   admin privileges, or to allow trusted users such as image analysts to carry
+   out tasks on behalf of all other users
+-  fixed color conversion to RGBA
+-  added support for exporting images in a plate as OME-TIFF
+-  improved creation of rendering settings for image without any stats e.g.
+   32bit images
+-  improved performance for moving large Plates
+-  fixed import bug in OMERO.insight
+-  fixed projection of images if the range of timepoints specified is not the
+   full range
+-  added support for transfering ownership of all the data of a given user
+   using CLI
+-  renamed "Reverse Intensity" command to "Invert" in image viewers
+-  added support for ImageColumn with Screen and Plate targets in the
+   populate_metadata script
+-  OMERO.web UI fixes:
+   * improved display of Plates and Wells
+   * fixed label position for Wells
+   * added the ability to display Image and Well metadata in the Tables 
+     section for the same Plate
+   * improved copy/paste of rendering settings workflow
+   * improved layout in left-hand panel
+   * fixed filtering issue
+   * added support for administrators with restricted privileges to create
+     Project/Dataset for other users
+   * rolled back the display of tables in the viewer
+
+Sysadmin changes include:
+
+-  added support for the creation of administrators with restricted privileges
+   in OMERO.web admin panel
+-  added method to create administrators with restricted privileges specifying
+   a password
+-  added specific installation instructions for Debian 9
+-  added configuration to limit queries that public users can do in OMERO.web
+-  created minimal NGINX configuration file that can be included in a fixed
+   file to allow custom NGINX options to be defined only once (e.g. SSL
+   options)
+-  installed django-redis by default
+-  CLI improvements and fixes:
+   * fixed admin plugin so ``cleanse`` can handle larger directories
+   * added to chown plugin ability to target all of given users' data
+   * adjusted handling of standard input 
+   * added infrastructure to load external CLI plugins
+   * dropped support for command ``admin ports``
+
+Developer updates include:
+
+-  added method to JavaGateway to manipulate admin privileges
+-  fixed issue with JSONP decorator
+-  removed SciPy dependency
+-  adjusted OMERO.blitz API to allow some query results to be cached
+-  added support to the Rendering engine to update a series of settings in one
```

Comment:
Should this be capitalised as "Rendering Engine" or "rendering engine"?  It looks inconsistent.